### PR TITLE
fixx language-errors

### LINKED ISSUE #183: tidy update dupe key in localization files
image

### LINKED ISSUE #184: fix `AutomaticFireIgnoredOnes` in localization files
should change to `AutomaticFireModifier`
in:
- es.json
- pt-br.json
- sv.json

ref: https://github.com/winks-vtt/yze-coriolis/pull/179

### LINKED ISSUE #186: Duplicate keys in en.json
Duplicate JSON keys result in invalid JSON file so localization software like Transifex throw an error:

> Duplicate string key ('YZECORIOLIS\.Talent') in line 207 

- https://github.com/winks-vtt/yze-coriolis/blob/185b55fdf7ecaa1219dbe5f3af4d3f1f6a0736ca/lang/en.json#L137
- https://github.com/winks-vtt/yze-coriolis/blob/185b55fdf7ecaa1219dbe5f3af4d3f1f6a0736ca/lang/en.json#L207

### LINKED ISSUE #189: Incorrect language reference
Desperation rolls refer to the language variable "YZECORIOLIS.RollSubstitute" but in the lang-files the variable is named "YZECORIOLIS.RollDesperation"

### PR DESCRIPTION
**Pull request by IrishWolf**
resolves #183
resolves #186
resolves #189
resolves #184

### CODE REVIEW

File: lang/es.json [hunk 1]
```diff
@@ -326,5 +326,5 @@
   "YZECORIOLIS.ItemDeleteConfirmationTitle": "Eliminar elemento",
   "YZECORIOLIS.ItemDeleteConfirmationContent": "¿Está seguro de que desea eliminar este elemento?",
   "YZECORIOLIS.AutomaticFire": "Fuego automático",
-  "YZECORIOLIS.AutomaticFireIgnoredOnes": "Número de 1 ignorados (máx. 2)"
-}
\ No newline at end of file
+  "YZECORIOLIS.AutomaticFireModifier": "Número de 1 ignorados (máx. 2)"
```
Comment (coderunner) [inline lang/es.json:329]: I think the meaning of this translation has changed. I don't think this text is relevant anymore. Google translate gives me '-2 modificador a tu tirada de ataque'

File: lang/sv.json [hunk 2]
```diff
@@ -383,5 +383,5 @@
   "YZECORIOLIS.ItemDeleteConfirmationTitle": "Ta bort elementet",
   "YZECORIOLIS.ItemDeleteConfirmationContent": "Är du säker på att du vill ta bort det här elementet?",
   "YZECORIOLIS.AutomaticFire": "Automatisk eld",
-  "YZECORIOLIS.AutomaticFireIgnoredOnes": "Antal ignorerade 1:or (max 2)"
-}
\ No newline at end of file
+  "YZECORIOLIS.AutomaticFireModifier": "Antal ignorerade 1:or (max 2)"
```
Comment (coderunner) [inline lang/sv.json:386]: Google translate again: `-2 modifierare till din attackrulle`

File: lang/pt-br.json [hunk 3]
```diff
@@ -383,5 +383,5 @@
   "YZECORIOLIS.ItemDeleteConfirmationTitle": "Excluir item",
   "YZECORIOLIS.ItemDeleteConfirmationContent": "Tem certeza de que deseja excluir este item?",
   "YZECORIOLIS.AutomaticFire": "Fogo automático",
-  "YZECORIOLIS.AutomaticFireIgnoredOnes": "Número de 1s ignorados (max 2)"
-}
\ No newline at end of file
+  "YZECORIOLIS.AutomaticFireModifier": "Número de 1s ignorados (max 2)"
```
Comment (coderunner) [inline lang/pt-br.json:386]: This could be `-2 modificadores em sua jogada de ataque`

**Comment by coderunner** [comment]
I'd suggest we adjust the translation for `AutomaticFireModifier` since the old key meaning (AutomaticFireIgnoredOnes) doesn't apply to the new key.

**Comment by IrishWolf** [comment]
> I'd suggest we adjust the translation for `AutomaticFireModifier` since the old key meaning (AutomaticFireIgnoredOnes) doesn't apply to the new key.

Yes. you are right. I didn't thought about that - it is pushed!

**Pull request merged**